plugins/commands/requirements/src/main/kotlin/RequirementsCommand.kt

```diff
@@ -174,12 +174,20 @@ class RequirementsCommand : OrtCommand(
             Modifier.isAbstract(it.modifiers) || it.isAnonymousClass || it.isLocalClass
         }.sortedBy { it.simpleName }.forEach {
             runCatching {
+                fun isBundledPlugin(type: String) = it.packageName.startsWith("org.ossreviewtoolkit.plugins.$type")
+
                 val kotlinObject = it.kotlin.objectInstance
 
                 var category = "Other tool"
                 val instance = when {
                     kotlinObject != null -> {
                         logger.debug { "$it is a Kotlin object." }
+
+                        when {
+                            isBundledPlugin("packagemanagers") -> category = "PackageManager"
+                            isBundledPlugin("versioncontrolsystems") -> category = "VersionControlSystem"
+                        }
+
```

Comment:
Have you considered extracting a `fun CommandLineTool.getCategory()` ? To speparate concerns a bit?

Reply:
Done, but slightly differently.